console/src/components/Widget/Widget.js

```diff
@@ -71,14 +71,16 @@ const WidgetBase = kind({
 				</Row>
 			);
 			default: return (
-				<Layout align="center center" orientation="vertical" {...rest}>
+				<Column {...rest}>
 					{!noHeader ? (
 						<Cell shrink component={CompactHeader} noExpandButton={noExpandButton} onExpand={onExpand} view={view}>
 							{title}
 						</Cell>
 					) : null}
-					{children}
-				</Layout>
+					<Column align="stretch center">
```

Comment:
Can't have a `Column` in a `Column`. Only `Cell` is allowed in `Layout`/`Column`/`Row`. Maybe instead just change Line 74 to `align="stretch center"` rather than `"center center"`.

Reply:
`align="stretch center"` won't work in this case to fit the need.
And what's the reason we can't have `Layout` in a `Layout`? Even if so, some of the `Compact*` components have either `Row` or `Column` as their children to the `Widget` component. For example https://github.com/enactjs/agate-apps/blob/master/console/src/components/CompactAppList/CompactAppList.js#L33 and https://github.com/enactjs/agate-apps/blob/master/console/src/components/CompactMusic/CompactMusic.js#L45
and so on.

Reply:
Just because other components are wrong doesn't mean that this should be wrong too. Those should be fixed as well.

A Layout should not be a direct child of another Layout because that's not how it's designed. `Layout` may only have `Cell`s as children, and `Cell` may only have `Layout` as their parent. Enforcing this relationship ensures that layouts behave predictably and reliably on old browser versions or with children with arbitrary sizing rules or with children who need to marquee or with parent containers who dynamically size. Even though you may not see an issue at the time with using a Layout in a Layout without a Cell in between, doesn't mean there won't be some quirk that shows up later. Older implementations of `flex` are extremely quirky and unpredictable without specific rules in place; newer implementations are only a little bit better.

I see your point about the need to have `Column` inside `Column`. I'd be satisfied with the current implementation simply wraps the inner `Column` with a `Cell`:
`<Column align="stretch center">` -> `<Cell><Column align="stretch center"></Cell>`